### ISSUE TITLE
Fails to build on Mac (OSX High Sierra 10.13.6, Go 1.13.5)

### ISSUE THREAD

**Issue by digitalronin** [sl1pm4t/k2tf#57]
I get the following error when trying to build from source:

```
$ make build
go build -v
get "k8s.io/client-go": found meta tag get.metaImport{Prefix:"k8s.io/client-go", VCS:"git", RepoRoot:"https://github.com/kubernetes/client-go"} at //k8s.io/client-go?go-get=1
go: github.com/terraform-providers/terraform-provider-kubernetes@v1.10.0 requires
        github.com/keybase/go-crypto@v0.0.0-20190523171820-b785b22cc757: invalid pseudo-version: does not match version-control timestamp (2019-04-16T18:20:11Z)
make: *** [build] Error 1

$ go version
go version go1.13.5 darwin/amd64
```

**Comment by fyaretskiy** [comment]
@digitalronin 
https://github.com/dambrisco/k2tf fixes it though his version is failing tests.

**Comment by sl1pm4t** [comment]
This should be fixed now by #58
